packages/demo/examples/08-network/data.js

```diff
@@ -0,0 +1,95 @@
+function getRandomID() {
+  return Math.floor(Math.random() * 1000000000);
```

Comment:
preferred alternative to `math.random()`, or you can install the `uuid` module https://stackoverflow.com/a/19964557/1933266